<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowBase</class>
 <widget class="QMainWindow" name="WindowBase">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1327</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>停电管理查询</string>
  </property>
  <property name="windowIcon">
   <iconset resource="outagequery.qrc">
    <normaloff>:/outagequery/images/outagequery.png</normaloff>:/outagequery/images/outagequery.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableWidget" name="m_powerCutTable">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/dbquery/images/bk.png);</string>
      </property>
      <column>
       <property name="text">
        <string>客户名称</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>客户类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>所属厂站</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>停电开始时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>停电结束时间</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>停电时长</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>停电开关</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>停电类型</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>馈线信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>区域信息</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>丢失负荷容量(kW)</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1327</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>280</width>
     <height>240</height>
    </size>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="m_tabWidget">
       <property name="tabShape">
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>厂站</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTreeWidget" name="m_stTreeWidget">
             <column>
              <property name="text">
               <string>厂站</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QToolButton" name="m_allStBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>所有厂站</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="m_stQueryBtn">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_1">
        <attribute name="title">
         <string>区域</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QTreeWidget" name="m_AreaTreeWidget">
             <column>
              <property name="text">
               <string>区域信息</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QToolButton" name="m_allAreaBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>所有区域</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="m_AreaQueryBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>馈线</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QTreeWidget" name="m_feederTreeWidget">
             <column>
              <property name="text">
               <string>馈线</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QToolButton" name="m_allfeederBtn">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>所有馈线</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="m_feederQueryBtn">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_read"/>
   <addaction name="action_Exit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="timetoolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="action_read">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/dbquery/images/print.png</normaloff>
     <normalon>:/outagequery/images/confirm.png</normalon>:/dbquery/images/print.png</iconset>
   </property>
   <property name="text">
    <string>读取数据</string>
   </property>
   <property name="toolTip">
    <string>读取数据</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="outagequery.qrc">
     <normaloff>:/outagequery/images/quit.png</normaloff>:/outagequery/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="outagequery.qrc"/>
 </resources>
 <connections/>
</ui>
